include/lbann/layers/learning/base_convolution.hpp

```diff
@@ -593,107 +484,97 @@ class base_convolution_layer : public learning_layer {
     const DataType zero = DataType(0);
     const DataType one = DataType(1);
     const int num_gpus = this->m_cudnn->get_num_gpus();
-    const int mini_batch_size = this->m_model->get_current_mini_batch_size();
-    const int effective_mini_batch_size = this->m_model->get_effective_mini_batch_size();
```

Comment:
Why has the effective mini-batch size been removed and substituted with just the mini-batch size?  We just fixed this so that it was using the effective mini-batch and not dividing by the number of models and the mini-batch size in two separate steps.

Reply:
`mini_batch_size`  has been set to the effective mini-batch size. This seemed cleaner to me, but I can see how it could be obfuscating.